formats/filtered/src/Filtered.php

```diff
@@ -164,6 +165,7 @@ protected function getResultText( SMWQueryResult $res, $outputmode ) {
 		$result = [];
 		while ( $row = $res->getNext() ) {
 			$result[ $this->uniqid() ] = new ResultItem( $row, $this );
+			usleep( 1 );
```

Comment:
> but next returns false too early. 
> but copy/pasting its content into my loop directly. No dice.

Okay, this sounds more of a 7.1 issue, since `mResults` is not a lazy object (something that fetches objects at event point) but a "static" list of entries, I cannot see any other vector that would cause such behaviour.